addon/components/nrg-lightbox-thumbnail/component.js

```diff
@@ -20,9 +25,23 @@ export default Component.extend({
     });
     this._super(...arguments);
   },
+  cloneObject(initialPhoto) {
+    if (initialPhoto.toJSON) {
+      return JSON.parse(initialPhoto.toJSON());
```

Comment:
If it has `toJSON`, you should not parse it since it should return a plain js object.

```suggestion
      return initialPhoto.toJSON();
```